flows/photometry.py

```diff
@@ -9,735 +9,830 @@
 
 import os
 import numpy as np
-from bottleneck import nansum, nanmedian, nanmax, allnan, replace
+from bottleneck import nansum, allnan, replace
+import sep
 from timeit import default_timer
 import logging
 import warnings
-from copy import deepcopy
 
-from astropy.utils.exceptions import AstropyDeprecationWarning
+from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning, ErfaWarning
 import astropy.units as u
 import astropy.coordinates as coords
-from astropy.stats import sigma_clip, SigmaClip, gaussian_fwhm_to_sigma
+from astropy.stats import sigma_clip, SigmaClip
 from astropy.table import Table, vstack
 from astropy.nddata import NDData
 from astropy.modeling import models, fitting
-from astropy.wcs.utils import proj_plane_pixel_area
+from astropy.wcs.utils import proj_plane_pixel_area, fit_wcs_from_points
 from astropy.time import Time
 
 warnings.simplefilter('ignore', category=AstropyDeprecationWarning)
```

Comment:
Is this warnings.simplefilter needed twice? (properly not)
The reason it was here originally was that photutils would throw warnings when importing... Is that still the case with the newer version?

Reply:
We need this for now yes, especially with our version fixing of Astropy/Photutils